lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -237,6 +238,28 @@ def _get_congestion_control_config(service_mconfig):
     return True
 
 
+def _get_converged_core_config(service_mconfig):
+    """
+    Retrieves enable_converged_core config value, it it does not exist
+    it defaults to True.
+    Args:
+        service_mconfig:
+
+    Returns: converged core flag
+    """
+    enable_converged_core = get_service_config_value(
+        'mme', 'enable_converged_core', None,
+    )
+
+    if enable_converged_core is not None:
+        return enable_converged_core
+
+    if service_mconfig.enable_converged_core is not None:
+        return service_mconfig.enable_converged_core
+
+    return True
```

Comment:
Default to false for backwards compatibility

Reply:
Will do that change.